### ISSUE TITLE
Port statistics.php to Rails

### ISSUE THREAD

**Issue by jfly** [thewca/worldcubeassociation.org#305]
@timhabermaas is working on this here: #175 (abandonded :cry:)

~~Related is #103~~ (no longer relevant).

**Comment by jfly** [comment]
@larspetrus says:

> The stats page is a BIG thing to reimplement for one person in one PR. I count 15 different sections.
>
> This makes it a great candidate to build incrementally. Like this:
>
> Make a new Rails page on, say, /results/statistics, that is not linked from the app. Implement a few of the simplest sections. Review, merge and deploy. Other people or the same person can later fill in more holes until one day we decide it's better than the old page and switch over.

I like this plan!

**Comment by AlbertoPdRF** [comment]
So, this one should be my next step after #4486 is merged, but before going ahead I think it's worthy to discuss about if we actually should be porting this page as is, only port a part of it, or just get rid of it altogether. Some key points that resulted from a conversation with @jonatanklosko:

- Does it make sense to have a set of arbitrary statistics on the WCA's website? People already do them on many other places, maybe just having a resources page linking to currently maintained statistics pages would be better?
- How should we decide which statistics make it to our website and which don't?
- Having stats always goes with a computation cost. I think the page could be way more manageable by diving it in tabs or something similar, but still, maybe we should consider if our resources could be better spent elsewhere?

What are other opinions on this?

**Comment by pedrosino** [comment]
I think the page is worth keeping, but maybe not all of it.

We could have some "more important" statistics and links to other resources as you said.

**Comment by jfly** [comment]
If we didn't have the statistics page, I would not be supportive of adding it. However, since we have it, it's harder to imagine removing it =( I think we need to be careful about linking to third party websites, because they can die/be wrong/have inappropriate content.

**Comment by AlbertoPdRF** [comment]
Yeah, that makes sense... Would you port it as is, or only part of it?

**Comment by viroulep** [comment]
Given we'll pretty much always prioritize other needed php ports (ie: last admin scripts), this is for sure gonna be the last thing we port (or drop).
I personally only visit that page occasionally, and found @jonatanklosko's website to be more exhaustive.

I do like the way it's split for the "Most *" sections.

Maybe something we could poll the staff on, since we do have polls!

**Comment by gregorbg** [comment]
Closing in favor of https://github.com/thewca/statistics
